src/clients/InsightsQueryClient.ts

```diff
@@ -171,6 +171,18 @@ class InsightsQueryClient {
     return response.data as Insights.InsightsQueryResponse<Insights.QueryGetPNLCompetitionLeaderboardResponse>;
   }
 
+  async LotteryCompetitionLeaderboard(
+    req: Insights.QueryGetLotteryCompetitionLeaderboardRequest
+  ): Promise<Insights.InsightsQueryResponse<Insights.QueryGetLotteryCompetitionLeaderboardResponse>> {
+    const queryParams = {
+      competitionId: req.competitionId,
+      ...(req.market && { market: req.market }),
+    };
+    const request = this.apiManager.path("competition/leaderboardlottery", {}, queryParams);
+    const response = await request.get();
+    return response.data as Insights.InsightsQueryResponse<Insights.QueryGetLotteryCompetitionLeaderboardResponse>;
+  }
```

Comment:
Do update `examples/query_insights.ts` with this function and test to make sure you receive the required output.